wondrous-app/components/Settings/TaskImport/GenericImportTaskModal/styles.tsx

```diff
@@ -0,0 +1,290 @@
+  }
+
+  &.MuiPaper-root > .MuiList-root {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+  }
+
+  &.MuiPaper-root > .MuiList-padding {
+    padding: 4px;
+  }
+`;
+
+export const GenericImportTaskModalSelectMenuItem = styled(MenuItem)`
+  && {
+    background: ${palette.grey87};
+    color: ${palette.white};
+    border-radius: 4px;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    padding: 10px;
+    transition: background 0.2s ease-out;
+
+    :hover {
+      background: ${palette.black92};
+    }
+  }
+`;
+
+export const GenericImportTaskModalSelectValueDisplay = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`;
+
+export const GenericImportTaskModalSelectValueDisplayText = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 19.14px;
+    color: ${(props) => (props.isActive ? palette.white : palette.grey57)};
+  }
+`;
+
+export const GenericImportTaskModalBodyExpandedViewWrapper = styled(Accordion)`
+  && {
+    background: ${palette.grey900};
+    margin-top: ${(props) => (props.expanded ? 0 : '-24px')};
+
+    & .MuiAccordion-region {
+      display: flex;
+      flex-direction: column;
+      gap: 26px;
+      box-shadow: none;
+    }
+
+    &.MuiPaper-root.MuiAccordion-root {
+      box-shadow: none;
+    }
+
+    &.MuiPaper-root::before {
+      display: none;
+    }
+  }
+`;
+
+export const GenericImportTaskModalBodyExpandedViewInvisibleState = styled(AccordionSummary)`
+  && {
+    display: none;
+  }
+`;
+
+export const GenericImportTaskModalFooter = styled.div`
+  background: ${palette.black97};
+  padding: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  gap: 18px;
+`;
+
+export const GenericImportTaskModalFooterButton = styled(Button)`
+  && {
+    background: ${(props) =>
+      props.disabled
+        ? 'transparent'
+        : props.isPrimary
+        ? `linear-gradient(
+      270deg,
+      ${palette.blue20} -5.62%,
+      ${palette.highlightPurple} 45.92%,
+      ${palette.highlightBlue} 103.12%
+    )`
+        : palette.grey85};
+
+    button {
+      font-family: ${typography.fontFamily};
+      font-weight: 600;
+      font-size: 15px;
+      line-height: 22.5px;
+      letter-spacing: -1%;
+      color: ${palette.white};
+      padding: 9px 24px;
+      border-radius: 1000px;
+      border: 1px solid;
+      border-color: ${(props) => (props.disabled ? palette.grey78 : props.isPrimary ? 'transparent' : palette.grey78)};
+      background: ${(props) => (props.isPrimary ? palette.background.default : palette.grey78)};
+      transition: background 0.2s ease-out;
+
+      :hover {
+        background: transparent;
+      }
+
+      :disabled {
+        background: transparent;
+        color: ${palette.grey78};
+        cursor: not-allowed;
+      }
+    }
+  }
+`;
+
+export const GenericImportTaskModalProgressSpinner = styled(CircularProgress)`
```

Comment:
you don't need write styled for this, use `<CircularProgress marginLeft="12px">`

Reply:
Apparently CircularProgress doesn't support marginLeft as a prop :/